### ISSUE TITLE
ios: No exact matches in call to instance method 'jsBundleURL' in ReactShareViewController.swift

### ISSUE THREAD

**Issue by xavax31** [meedan/react-native-share-menu#209]
Hi,
I tried to implement custom view following the doc, but I get error while building:
<img width="405" alt="Capture d’écran 2022-09-07 à 12 26 45" src="https://user-images.githubusercontent.com/4051022/188856521-1f0281a7-da43-427a-b79d-2e9c17ed65e3.png">
<img width="1016" alt="Capture d’écran 2022-09-07 à 12 26 56" src="https://user-images.githubusercontent.com/4051022/188856559-fba7075b-e1b0-4264-af80-ac2d8881212f.png">

It works running the example (after fix error by adding in podfile: 'useFlipper!({ 'Flipper-Folly' => '2.5.3', 'Flipper' => '0.87.0', 'Flipper-RSocket' => '1.3.1' })
My project targets ios 12 rather than ios 10 as in example, but if i change to 10, I get an error saying that RNShareMenu need ios 11.
I use exactly same index.share.js and Share.js that the example.

Note: I'm on Macbook M1

Any idea?

**Comment by o-baranova** [comment]
Same problem appeared after switching from "react-native": "0.64.3" => "react-native": "0.69.5".
Have you found a solution?

**Comment by xavax31** [comment]
Yes, after some research, I forked the package and made this change:  
https://github.com/xavax31/react-native-share-menu/commit/20917a84d4dea48d14fd84902e61b01548e4861b

**Comment by ice-cap0** [comment]
Same issue when implementing custom iOS share component. 

Anybody found a solution that doesn't require a patch ? 

BTW no reason to use a fork anymore now that [`patch-package`](https://github.com/ds300/patch-package) exists

**Comment by JesusTectronic** [comment]
In the ReactShareViewController file change your code for this

```js
return RCTBundleURLProvider.sharedSettings()?
      .jsBundleURL(forBundleRoot:"index.share")
      //.jsBundleURL(forBundleRoot: "index.share", fallbackResource: nil)
```

**Comment by AleksandarVasicBlueGrass** [comment]
Removing ? worked for me
